openupgradelib/openupgrade.py

```diff
@@ -1,26 +1,10 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    This module copyright (C) 2011-2013 Therp BV (<http://therp.nl>)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2011-2020 Therp BV <https://therp.nl>.
```

Comment:
This copyright is incomplete, although I understand the abbreviation. Please add at least:

```
Copyrigh 2016-2020 Tecnativa - Pedro M. Baeza
```

I will study later other contributors.

Reply:
I will add you. From git blame I see a lot op people worked on this. Especially Jairo Lopez. Also there were a lot of contributions from Stefan after he left Therp, then other people contributed as well.

Note that I only changed the format of the copyright and the enddate.

Reply:
Can we just assign copyright to OCA and update AUTHORS.rst?

Reply:
Here is the updated list of contributors.

```
* Alexandre Fayolle
* Alexis de Lattre
* Arthur Vuillard
* Benoit Guillot
* Bogdan Stanciu
* Christoph Giesel
* Daniel Reis
* David Arnold
* David Béal
* David Vidal
* Dimitrios T. Tanis
* Dmytro Katyukha
* Ernesto Tejeda
* Florent Xicluna
* Florian Dacosta
* Graeme Gellatly
* Hector Villarreal Ortega
* Hervé Martinet
* Holger Brunn
* Ivàn Todorovich
* Jairo Llopis
* Jordi Riera
* Juan Pablo Arias
* Julien Coux
* Katherine Zaoral
* Kay Häusler
* Manuel Vázquez Acosta
* Miku Laitinen
* Miquel Raich
* Moisés López
* Paulius Sladkevičius
* Pedro M. Baeza
* Petar Najman
* Robert Rübner
* Ronald Portier
* Sandy Carter
* Sebastien Alix
* Silvija Butko
* Stefan Rijnhart
* Stephane Le Cornec
* Stéphane Bidoul
* Sylvain LE GAL
* Tom Blauwendraat
* Yann Bongiovanni
* Yuri Quintana
```

Reply:
Updated AUTHORS.rst by rebasing.